framework/contrib/gtest/gtest/gtest.h

```diff
@@ -21198,4 +21199,9 @@ inline int RUN_ALL_TESTS() {
   return ::testing::UnitTest::GetInstance()->Run();
 }
 
+// Restore warnings to other classes (see top of file for info on ignored warnings)
+#ifdef __clang__
+#pragma clang diagnostic pop
+#endif
```

Comment:
@lindsayad I think this and Line 57 addresses your previous comment about this ignore impacting other unit test objects. The "push/pop" pragmas are the same used in the libmesh `ignore_warnings.h` and `restore_warnings.h` to isolate what is ignored.